src/main/app/case/to-api-format.ts

```diff
@@ -149,7 +149,7 @@ const addressPrivateYesNoOrNull = (value: YesOrNo) => {
   if (!value) {
     return null;
   }
-  return value === YesOrNo.YES ? ConfidentialAddress.KEEP : ConfidentialAddress.SHARE;
+  return value;
```

Comment:
I think you could get rid of this method now and just use the standard YesOrNo conversion

Reply:
done https://github.com/hmcts/nfdiv-frontend/pull/1199/commits/fe198e567c09e50c9a7f84aca544f4ef765a81ea